libp2p/transport/webrtc/async_bridge.py

```
@@ -143,7 +143,17 @@ async def close_peer_connection(self, peer_connection: RTCPeerConnection) -> Non
     async def close_data_channel(self, data_channel: RTCDataChannel) -> None:
         """Close data channel with proper async bridging"""
         try:
-            await aio_as_trio(data_channel.close)
+            close_attr = getattr(data_channel, "close", None)
+            if close_attr is None:
+                logger.debug("Data channel has no close attribute")
+            else:
+                result = close_attr()
+                if hasattr(result, "__await__"):
+                    try:
+                        await aio_as_trio(result)

```

This exception handling in `await aio_as_trio(result)` catches all exceptions but then tries `close_attr()` again - which might mask underlying errors,  right? 
we should log the specific exception before falling back.